ci/scripts/build.sh

```diff
@@ -59,3 +63,11 @@ buildkite-agent artifact upload risedev-dev-"$profile"
 buildkite-agent artifact upload risingwave_regress_test-"$profile"
 buildkite-agent artifact upload ./sqlsmith-"$profile"
 buildkite-agent artifact upload ./compaction-test-"$profile"
+
+echo "--- Build Java connector node"
+git clone https://"$GITHUB_TOKEN"@github.com/risingwavelabs/risingwave-connector-node.git
```

Comment:
Just some random thoughts: So we're always using the HEAD of the main branch? If there're some issues on this commit, will it lead all PRs in RisingWave repo fail to pass the CI? Do we have a plan for using something like nightly release? 🤔

Reply:
Good point. For now, I think we can tag a released version on the Java connector node and this script will clone a specified tag.